pyxcli/events/events.py

```diff
@@ -24,7 +24,7 @@
 
 import json
 from socket import getfqdn
-from bunch import Bunch
+from munch import Munch
```

Comment:
Why Munch? 
A Replacement like this requires extensive testing. 
What's the benefit of this?

Reply:
I don't believe that Bunch is still supported, and it is incompatible with Python 3.x. All of my tests failed with Bunch in Python 3. It appears that Infinidat forked Bunch and created Munch.

https://github.com/Infinidat/munch